### PR TITLE
DDL Versioning

### PR DESCRIPTION
**Pull request by AdamHawtin**
# Motivation and Context
We've added a new schema, the rebuild and destroy scripts should include it

# What has changed
* Use loop in rebuild script
* Add ddl_version script

# How to test?
Run the scripts against a GCP database, check the schema in created/destroyed correctly. 

# Links
https://trello.com/c/fAB7azyN/403-ddl-versioning-13

### CODE REVIEW

**Comment by AdamHawtin** [comment]
[DDL versioning (13)](https://trello.com/c/fAB7azyN/403-ddl-versioning-13)

File: groundzero/rebuild_from_ground_zero.sh [hunk 1]
```diff
@@ -1,32 +1,23 @@
...
-mv header_footer_temp.txt actionv2.sql
-
-echo "begin transaction;" > header_footer_temp.txt
-cat casev2.sql >> header_footer_temp.txt
-echo "commit transaction;" >> header_footer_temp.txt
-rm casev2.sql
-mv header_footer_temp.txt casev2.sql
+for SCRIPT_NAME in actionv2.sql casev2.sql uacqid.sql ddl_version.sql
+do
+  rm $SCRIPT_NAME
```
Comment (liamtoozer) [inline groundzero/rebuild_from_ground_zero.sh:8]: Probably don't need to remove this first as it's removed earlier on?

Comment (AdamHawtin) [reply groundzero/rebuild_from_ground_zero.sh:8]: @liamtoozer I actually meant to delete those first `rm`s so the script only has to have the script names in one place, I'll fix that

**Review by liamtoozer — APPROVED** [review]

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

File: groundzero/rebuild_from_ground_zero.sh [hunk 2]
```diff
@@ -1,32 +1,18 @@
...
-echo "begin transaction;" > header_footer_temp.txt
-cat actionv2.sql >> header_footer_temp.txt
-echo "commit transaction;" >> header_footer_temp.txt
-rm actionv2.sql
-mv header_footer_temp.txt actionv2.sql
+  echo "begin transaction;" > header_footer_temp.txt
+  cat $SCRIPT_NAME >> header_footer_temp.txt
+  echo "commit transaction;" >> header_footer_temp.txt
+  rm $SCRIPT_NAME
+  mv header_footer_temp.txt actionv2.sql
```
Comment (ryangrundy7) [inline groundzero/rebuild_from_ground_zero.sh]: Don't think this is meant to still be in here?

**Review by ryangrundy7 — CHANGES_REQUESTED** [review]
Don't think the `mv` is meant to be there

File: groundzero/rebuild_from_ground_zero.sh (hunk 2 above)
Comment (AdamHawtin) [reply groundzero/rebuild_from_ground_zero.sh]: Yup, missed one of the script names, good catch

**Review by ryangrundy7 — APPROVED** [review]

**Pull request merged**